### ISSUE TITLE
Terminal

### ISSUE THREAD

**Issue by xandaaah** [Flow-Works/FlowOS-OLD#196]
**Is your feature request related to a problem? Please describe.**
If this webOS has a kernel(ish), what is the point of not having a terminal? This would be useful.
**Describe the solution you'd like**
The shell is similar to Bash, and the terminal is similar, to well, like conhost (CMD is the shell, conhost is the terminal emu), or Konsole.

**Describe alternatives you've considered**
...

**Comment by httphypixelnet** [comment]
Yeah I'm slowly but surely trying to work on that but if one of the actual devs can do it that would be way better.
